From 8f93a36bac4c9088b20ea6ee586391ef2c7b996b Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 3 Nov 2021 16:35:51 +0100
Subject: [PATCH 275/529] ARM: dts: stm32: add simple-framebuffer on
 stm32mp135f-dk

Add node to support the simple-frambuffer. By default, the node must be
disabled.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
Change-Id: I37bc4c7494a3fc2cdfc5a2b7257b34f38aa5ffa1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228341
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/280106
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/boot/dts/stm32mp135f-dk.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

--- a/arch/arm/boot/dts/stm32mp135f-dk.dts
+++ b/arch/arm/boot/dts/stm32mp135f-dk.dts
@@ -28,6 +28,15 @@
 
 	chosen {
 		stdout-path = "serial0:115200n8";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		framebuffer {
+			compatible = "simple-framebuffer";
+			clocks = <&rcc LTDC_PX>;
+			status = "disabled";
+		};
 	};
 
 	memory@c0000000 {
